en/12_Project_Governance/05_Release_policy.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Release policy
+summary: Outline the release lifecycle and support commitment for Silverstripe CMS
+icon: code-branch
+---
+
+# Silverstripe CMS release policy
+
+This document outlines the release policy for Silverstripe CMS. It applies to all [commercially supported modules](/project_governance/supported_modules/).
+
+Community modules are not covered by this policy. Modules in the silverstripe GitHub organisation that are not commercially supported are updated on a best effort basis.
+
+> [!NOTE]
+> Refer to our [definition of public API](/project_governance/public_api/).
+
+The purpose of this policy is to provide clear guidance on the types of releases, their cadence, scope, and support lifecycle, ensuring predictability and stability for our users and developers.
+
+## Release types and cadence
+
+Silverstripe CMS adheres to [Semantic versioning](https://semver.org/) for its release philosophy.
+
+### Major releases `(x.0.0)`
+
+Major releases include architectural updates, allow breaking changes and major dependency changes.
+
+- **Purpose:** To deliver breaking changes and architectural updates that cannot be shipped in minor releases due to backward compatibility constraints.
+
+- **Backward compatibility:** Major releases are **not backward compatible**.
+
+- **Frequency:** New Silverstripe CMS major versions are released every **two years**.
+
+- **Tagging period:** New major releases are typically tagged as stable between **April and June** of their release year.
+
+- **Scope:** This policy applies to all Silverstripe CMS commercially supported modules. At launch, the list of commercially supported modules for that specific major release line will be explicitly defined.
+
+- **Pre-release period:** A stable major release is always preceded by a **beta period of at least three months** to allow for extensive testing and feedback.
+
+- **Dependency alignment:** At launch, Silverstripe CMS major releases aim to:
+
+  - Support all PHP versions currently in full support.
+  - Have all PHP and JavaScript dependencies on their latest stable versions.
+
+- **Upgrade path:** Silverstripe CMS commits to providing a clear upgrade path to new major releases, including comprehensive documentation on all breaking changes and migration guides.
+
+### Minor releases `(x.y.0)`
+
+Minor releases provide ongoing value and improvements within a stable major release line. They are backwards compatible within the major release line and designed to be easy to upgrade to.
```

Comment:
```suggestion
Minor releases provide ongoing value and improvements within a stable major release line. They are backwards compatible within the major release line and deliver new features and new public API in a backward compatible way.
```
Let's not use the "easy" word.
Suggested wording comes from the existing policy.